tests/preprocessing/test_atoms_to_graphs.py

```diff
@@ -17,13 +18,7 @@ def atoms_to_graphs_internals(request):
         format="json",
     )
     test_object = AtomsToGraphs(
-        max_neigh=12,
-        radius=6,
-        dummy_distance=7,
-        dummy_index=-1,
-        r_energy=True,
-        r_forces=True,
-        r_distances=True,
+        max_neigh=12, radius=6, r_energy=True, r_forces=True, r_distances=True,
```

Comment:
Make sure to update this in the other tests as well; all of those are probably broken now because we dropped `dummy_*`.

[We really need a way to automatically run tests on every push :)]